Trappist/src/Promact.Trappist.Repository/TestConduct/TestConductRepository.cs

```diff
@@ -340,20 +364,36 @@ public TestConductRepository(TrappistDbContext dbContext, ITestsRepository testR
 
                 if (isAnsweredOptionCorrect)
                 {
-                    marks += testAttendee.Test.CorrectMarks;
+                    correctMarks += testAttendee.Test.CorrectMarks;
                 }
                 else
                 {
-                    marks += testAttendee.Test.IncorrectMarks;
+                    correctMarks -= testAttendee.Test.IncorrectMarks;
                 }
             }
-
+            totalMarks = correctMarks;
             var report = await _dbContext.Report.SingleOrDefaultAsync(x => x.TestAttendeeId == testAttendeeId);
-            report.TotalMarksScored = (double)marks;
+            report.TotalMarksScored = (double)totalMarks;
             report.Percentage = (report.TotalMarksScored / (double)fullMarks) * 100;
             _dbContext.Report.Update(report);
             await _dbContext.SaveChangesAsync();
         }
+
+        private async Task GetTimeTakenByAttendeeAsync(int attendeeId)
+        {
+            var testAttendee = await _dbContext.TestAttendees.FirstOrDefaultAsync(x => x.Id == attendeeId);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused 'testAttendee' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1481)